packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -228,8 +228,10 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
       this.paymentChannelClient.channelCache[channelId].beneficiaryBalance
     );
     // infer payment using update balance and previously stored balance
-    const payment = newSeederBalance.sub(
-      bigNumberify(this.peersList[infoHash][peerId].seederBalance)
+    const payment = bigNumberify(
+      this.peersList[infoHash][peerId]
+        ? newSeederBalance.sub(bigNumberify(this.peersList[infoHash][peerId].seederBalance))
+        : 0
```

Comment:
This assignment is too long and hard to read. maybe store `this.peersList[infoHash][peerId]` in a var and avoid using `bigNumberify` twice.